fluentd/run.sh

```diff
@@ -167,12 +188,13 @@ OPS_COPY_PASSWORD="${OPS_COPY_PASSWORD:-$ES_COPY_PASSWORD}"
 export OPS_COPY_HOST OPS_COPY_PORT OPS_COPY_SCHEME OPS_COPY_CLIENT_CERT \
        OPS_COPY_CLIENT_KEY OPS_COPY_CA OPS_COPY_USERNAME OPS_COPY_PASSWORD
 
-if [ "$ES_COPY" = "true" ] ; then
+if [ "$ES_COPY" = "true" -a "${USE_MUX_CLIENT:-}" != "true" ] ; then
     # user wants to split the output of fluentd into two different elasticsearch
     # user will provide the necessary COPY environment variables as above
     cp $CFG_DIR/{openshift,dynamic}/es-copy-config.conf
     cp $CFG_DIR/{openshift,dynamic}/es-ops-copy-config.conf
 else
+    echo "WARNING: When USE_MUX_CLIENT is true, logs are forwarded to MUX; COPY won't work with it."
```

Comment:
Ahhh, thanks again, @portante .
The logic should be fine.  (MUX_CLIENT_MODE and USE_MUX_CLIENT are not passed to mux.  So, if ES_COPY is true, the copy config files are set to the expected location /etc/fluent/configs.d/dynamic on the mux pod.)  B*U*T, the WARNING should be issued only when ES_COPY is true...  The fix has been pushed.